dronekit/__init__.py

```diff
@@ -254,6 +252,73 @@ def __init__(self, distance, voltage):
     def __str__(self):
         return "Rangefinder: distance={}, voltage={}".format(self.distance, self.voltage)
 
+class Version(object):
+    """
+    The version number in a few different formats. To get it in a human-readable
+    format, just print `vehicle.version.number`. Or read the version in its four
+    parts: major, minor, patch, release_type, by calling `vehicle.version.number.major`, etc.
+
+    To check if the version is a stable release, check `vehicle.version.number.is_stable()`
+    """
+    def __init__(self, raw_version, autopilot_type, vehicle_type):
```

Comment:
Not every version will send the message. We use the convention that `None` is used by default for attribute values on object creation, and these are then populated with values on the first message received. That way it is obvious to users of the API when the attribute isn't supported.